client/ayon_maya/api/lib.py

```diff
@@ -4320,6 +4322,8 @@ def create_rig_animation_instance(
             pre_create_data={"use_selection": True}
         )
 
+    cmds.lockNode(f"*{namespace}", lock=lock_set_on_load)
```

Comment:
This does more than lock the set. This has a high change of matching way more than intended. We should **explicitly** target the one set you want to lock to do this correctly.